toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -69,8 +69,8 @@ func buildDracutModule(dracutModuleName string, dracutDriverName string, imageCh
 	// Check if the dracut module configuration file already exists.
 	if _, err := os.Stat(dracutConfigFile); os.IsNotExist(err) {
 		lines := []string{
-			"add_dracutmodules+=\"" + dracutModuleName + "\"",
-			"add_drivers+=\"" + dracutDriverName + "\"",
+			"add_dracutmodules+=\" " + dracutModuleName + " \"",
```

Comment:
nit - you may want to add a comment here.

Reply:
Fixed, thanks.